.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Checks
+
+on: pull_request
+
+jobs:
+  pr-checks:
+    runs-on: macos-latest
+
+    steps:
+      - name: Clone the Repository
+        uses: actions/checkout@v3
+
+      - name: Install Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: Lint
+        run: deno lint
```

Comment:
Only linting in this PR. Future checks will follow in subsequent PRs. For this PR, we should just ensure that the structure of the file is appropriate and mutually understood.